docs/reference/projectyml.rst

```diff
@@ -50,66 +50,122 @@ The code snippet below provides an example of a complete ``project.yml`` file:
     freeze_recursive: true
     freeze_operator: ~=
     pip:
-      use_config_file: false
-      index_urls:
-        - https://pypi.org/simple/
+        index-url: https://pypi.org/simple/
+        extra-index-url: []
+        pre: false
+        use-system-config: false
 
 
 .. _specify_location_pip:
 
+Configure pip index
+-------------------
 
-Specify locations from where V2 modules will be installed
----------------------------------------------------------
-This section explains how to configure your project in order to download v2 modules from any python package repository.
-By default, a project created using the :ref:`project-creation-guide` is configured to install packages from ``https://pypi.org/simple/``. There are multiple ways
-to change this behaviour.
+This section explains how to configure a project-wide pip index. This index will be used to download v2 modules and v1
+modules' dependencies.
+By default, a project created using the :ref:`project-creation-guide` is configured to install packages from ``https://pypi.org/simple/``.
+The :class:`~inmanta.module.ProjectPipConfig` section of the project.yml file offers options to configure this behaviour.
 
-Using pip config file
+pip.use-system-config
```

Comment:
I figured the class documentation for `ProjectPipConfig` was enough for these. Maybe I can add some explicit sections that point to `ProjectPipConfig` ?

Reply:
You make a good point. I don't think explicit sections are required. But perhaps where you now have "The :class:`~inmanta.module.ProjectPipConfig` section of the project.yml file offers options to configure this behaviour.", we follow that up by stating that some of those options are explained in more detail below? This would make it more explicit that what follows is not an exhaustive list of all options. Wdyt?

Reply:
Great idea :+1: